### ISSUE TITLE
Newly Installed Steam Art Manager Fails to Open

### ISSUE THREAD

**Issue by baltimoresports** [Tormak9970/Steam-Art-Manager#137]
**Describe the bug**
Installed Steam Art Manager on a newly imaged Windows 11 PC and when I click on the executable I see a brief flash of the program and it crashes without an error. 

**To Reproduce**
1. Install with Chocolatey or MSI
2. Click Executable 

**Expected behavior**
Should open

**Logs:**
No log or even folder created: `C:\Users\USER\AppData\Roaming\dev.tormak.steam-art-manager\logs\core.log`

**System information (please complete the following information):**
 - OS: Windows 11
 - Version 21H2

**Comment by Tormak9970** [comment]
@baltimoresports Do you have Steam installed? If so, have you logged in to steam?

**Comment by baltimoresports** [comment]
Yes to both, but I wonder if I opened your app before signing in.

**Comment by Tormak9970** [comment]
> Yes to both, but I wonder if I opened your app before signing in.

Alright. Can you try it again now?

**Comment by baltimoresports** [comment]
Works now sir! Thank you.
